ush/gefs_atmos_extractvars.sh

```diff
@@ -0,0 +1,102 @@
+#! /usr/bin/env bash
```

Comment:
Replace all `gefs` in this script with `${RUN}`.  See comment below.
And, rename this script (and the other ocnice, wav) to `atmos_extractvars.sh`, etc.

Reply:
Done. Removed "gefs_" from ush scripts and replaced `gefs` with `RUN`.